### DISCUSSION TITLE
Consider moving to BullMQ

### DISCUSSION THREAD

**Discussion by dwene** [medusajs/medusa#2984] [Feature Requests]
Bull has been moved to maintenance mode (https://github.com/OptimalBits/bull#-news-and-updates), and the bull team has completely rewritten Bull as BullMQ in typescript (https://docs.bullmq.io/). 

They fixed a lot of issues in Bull (stalled jobs, concurrency), which I think would really help the event-bus service thrive and not get blocked by long running jobs.

**Comment by olivermrbl** [comment]
Thank you for reporting this one @dwene. We will definitely investigate a migration to BullMQ, and move on it in the near future :)
